samples/SkiaMultiplatformSample/src/wasmJsMain/kotlin/org/jetbrains/skiko/sample/App.wasm.kt

```diff
@@ -1,5 +1,21 @@
 package org.jetbrains.skiko.sample
 
+import kotlinx.browser.document
+import org.jetbrains.skiko.GenericSkikoView
+import org.jetbrains.skiko.SkiaLayer
+import org.w3c.dom.HTMLCanvasElement
+
 fun main() {
     runApp()
 }
+
+internal fun runApp() {
+    val skiaLayer = SkiaLayer()
+    skiaLayer.onContentScaleChanged = { scale -> println(scale) }
```

Comment:
is this intended?  I mean, are we leaving this for debug purposes?

Reply:
It's just for debug purpose. And it's not new code, it's copy-pasted from jsWasmMain common